public/app/assetlibrary/assetLibraryService.js

```diff
@@ -27,22 +27,28 @@
 
   'use strict';
 
-  angular.module('collabosphere').service('assetLibraryService', function($state, analyticsService, utilService) {
+  angular.module('collabosphere').service('assetLibraryService', function(analyticsService, deepLinkId, utilService, $state) {
 
-    // Get the parent window's URL. In case any Collabosphere data is present,
-    // restore the state to allow for deep linking to an asset or asset library search
-    if (window.parent) {
-      utilService.getParentUrlData(function(data) {
-        if (data.asset) {
-          var assetId = parseInt(data.asset, 10);
+    var assetViewRedirect = function(assetId) {
+      // Track the asset deep link
+      analyticsService.track('Deep link asset', {
+        'asset_id': assetId,
+        'referer': document.referrer
```

Comment:
@johncrossman I would say we should fix this spelling, but I suppose the misspelling is also configured on the Mixpanel side.